### PR TITLE
feat(cmd): .spad command

### PR DESCRIPTION
**Pull request by Lauvaan**
## Description

A command that links to the SPAD.neXt discord server so we can redirect users easier to their server, so that users can get support regarding the spad program there.

## Test Results

Unable to test, away from home, only have chromebook and github.dev to work with.

### CODE REVIEW

File: src/commands/index.ts [hunk 1]
```diff
@@ -340,6 +341,8 @@ const commands: BaseCommandDefinition[] = [
     taf,
     remoteEfb,
     brakeCheck,
+    spad,
+
```
Comment (pdellaert) [inline src/commands/index.ts:345]:
```suggestion
```
Should remove the empty line :)

**Review by pdellaert — CHANGES_REQUESTED** [review]
Great work, one tiny tiny tiny cleanup :)

**Review by pdellaert — APPROVED** [review]

**Comment by benw202** [comment]
Just a quick on @Lauvaan, is the discord link a 'known permanent' link from them?

**Comment by Lauvaan** [comment]
> Just a quick on @Lauvaan, is the discord link a 'known permanent' link from them? 

No, I just made the link myselves, changed the setting to be permanent tho

**Comment by benw202** [comment]
> > Just a quick on @Lauvaan, is the discord link a 'known permanent' link from them? 
> 
> No, I just made the link myselves, changed the setting to be permanent tho

Ahh no worries, generally we get a server to make a link for us so they don't purge links, which can happen even if permanent. Straks has updated it with the link used on their website - https://github.com/flybywiresim/discord-bot/pull/477

**Pull request merged**